pkg/cmd/app.go

```diff
@@ -422,6 +429,7 @@ func startDCGMExporter(c *cli.Context) error {
 
 		// Only validate prerequisites if not disabled.
 		if !config.DisableStartupValidate {
+			config.FailOnNVMLInitError = true // enable failure on NVML Init Error
```

Comment:
@glowkey this means that if DisableStartupValidate is false, meaning we want startup validating, we will exit on NVML init error

if DisableStartupValidate is true, meaning we want to start up without validating rules (and now, nvml init errors), we can proceed with FailOnNVMLInitError = false (default).

this means users can also decide to enable DisableStartupValidate, but also enable FailOnNVMLInitError, so no rules are validated via prerequisites, but the exporter will bail if nvml init fails.

Reply:
would we rather this be enforced by DisableStartupValidate? the only disadvantage there is that people cant decide they want to skip validating the rules AND also bail on nvml init error.